ext/js/pages/settings/anki-controller.js

```diff
@@ -762,7 +762,8 @@ class AnkiCardController {
         const ELEMENT_NODE = Node.ELEMENT_NODE;
         const container = this._ankiCardFieldsContainer;
         if (container !== null) {
-            for (const node of [...container.childNodes]) {
+            const childNodesFrozen = [...container.childNodes];
```

Comment:
The linter was previously trying to convert this loop to just `for (const node of container.childNodes)` but I explicitly didn't want that because this list is live and elements are being removed from it.